website/docs/r/transit_secret_backend_key.html.md

```diff
@@ -53,6 +53,8 @@ The following arguments are supported:
 
 * `min_encryption_version` - (Optional) Minimum key version to use for encryption
 
+* `auto_rotate_interval` - (Optional) Amount of time the key should live before being automatically rotated. A value of 0 disables automatic rotation for the key
```

Comment:
nit: fold long lines exceeding 100 columns wide

Reply:
Fixed in edc6e4c